src/game/area.c

```diff
@@ -120,8 +131,8 @@ u32 get_mario_spawn_type(struct Object *o) {
     const BehaviorScript *behavior = virtual_to_segmented(0x13, o->behavior);
 
     for (i = 0; i < 20; i++) {
-        if (D_8032CE9C[i] == behavior) {
-            return D_8032CEEC[i];
+        if (WARP_BEHAVIOR_SPAWN_TABLE[i] == behavior) {
+            return SPAWN_TYPE_FROM_WARP_BHV[i];
```

Comment:
These variable names should be camelCase; UPPER_SNAKE_CASE denotes a macro

Reply:
sounds good, I'll change it

Reply:
These vars have been camel cased!

Reply:
and now prefixed with "s"